.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: python
 sudo: false
 python:
-    - 2.6
     - 2.7
+    - 3.5
+    - 3.6
```

Comment:
Since we have a coverage tox environment, can we get coveralls added here? 

Can we enable pip caching?

(And maybe ditch tox-travis/tox)?

Reply:
What's a good package to cargo-cult the current .travis-ci.yml best practices from?

Reply:
I often copy [zope.viewlet](https://github.com/zopefoundation/zope.viewlet/blob/master/.travis.yml):
```yaml
language: python
sudo: false
python:
    - 2.7
    - 3.4
    - 3.5
    - 3.6
    - pypy
    - pypy3
install:
    - pip install -U pip setuptools
    - pip install -U coverage coveralls
    - pip install -U -e .[test]
script:
    - coverage run -m zope.testrunner --test-path=src
after_success:
    - coveralls
notifications:
    email: false
cache: pip
```
(Of course that leaves off `--auto-color`.)

Reply:
`--auto-...` is overkill for Travis, IMHO, we *know* they give us a pty.  I like `zope-testrunner ... -pvc`.

Reply:
Do you know if the `before_cache: rm -f $HOME/.cache/pip/log/debug.log` I snarfed from zope.vocabularyregistry is no longer necessary?

Reply:
pip now seems to touch a bunch of files in its cache directory when it runs (probably something to do with the up-to-date check?) so removing that file no longer prevents a new cache archive from being created.

Reply:
I say that, but when I re-ran this build, it didn't write a new archive:
```
$ rm -f $HOME/.cache/pip/log/debug.log
store build cache
    nothing changed, not updating cache
```

Looking at some other projects, it's hard to tell exactly when pip will touch files. Here's the output for a different projects where only `six` was downloaded and updated.
```
0.01s$ rm -f $HOME/.cache/pip/log/debug.log
cache.2
store build cache
0.01s
4.53schange detected (content changed, file is created, or file is deleted):
/home/travis/.cache/pip/http/0/5/1/d/d/051ddbcfada687fb27df4b43966167a67f6706da37bcbb89d2838836
/home/travis/.cache/pip/http/0/a/8/f/a/0a8faabd212d81beff3ad0e11f3e4746188c0ad05c9190218de2e48a
/home/travis/.cache/pip/http/0/e/7/b/9/0e7b99547a0247a24df2175f35e166cf96f0194417e99b4851dbf306
/home/travis/.cache/pip/http/1/1/4/a/4/114a41049557a84c55958c18d02d44eb147fadc5d401e269e3d7ea8e
/home/travis/.cache/pip/http/1/c/f/2/e/1cf2ec08b116d0e8a4bd0c1c6441595628f3d6f6275c7d9cb54fb0c2
/home/travis/.cache/pip/http/2/6/6/b/d/266bdf220752466a969e134263f71564e77e2b9b41ad250878d8f5d8
/home/travis/.cache/pip/http/2/b/9/3/c/2b93c2d17b7b3f0ea9392df02b2eaa5b41f0eefacc87dc903a847b70
/home/travis/.cache/pip/http/2/f/1/0/6/2f10635579eda75bec25d08978dec5575ccd2fa532b1f347d21ec0a1
/home/travis/.cache/pip/http/3/7/2/f/2/372f280a7b4ee088e9327ee15a8c8095522419e73ce895ede6016b18
/home/travis/.cache/pip/http/3/a/f/3/a/3af3addf06e983a6c02f46e7bea70c221d3ff95bf1418fa6da354e14
/home/travis/.cache/pip/http/4/a/1/1/8/4a
...
```